unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDModel.cs

```diff
@@ -4,7 +4,7 @@
 
 public class AvatarEditorHUDModel
 {
-    public WearableItem bodyShape;
+    public WearableItem bodyShape = new WearableItem { id = "NOT_LOADED" };
```

Comment:
Why are we setting this id? For code clarity?

Reply:
yeah, and it will log in some cases, so it it clear